packages/generated-react/dist/example-TodoService_connectquery.ts

```diff
@@ -50,7 +50,7 @@ export const addTodo = createQueryService({
  *
  * @generated from rpc buf.connect.demo.example.v1.TodoService.Delete
  */
-export const delete$ = createQueryService({
+export const delete = createQueryService({
```

Comment:
uh oh.  what's going on here.  this is invalid syntax in javascript.  when I did a `pnpm all` it overwrote these, fixing this as well as the bad import at the top that's missing `@bufbuild/`.  I'd LOVE to understand how this happened.  so strange.

Reply:
I see above that it (somehow?) ran with 0.2.1 instead of 0.2.2.  not sure if/how that's related.

Reply:
Hmm, weird. Yeeeah. All i did was run `pnpm generate` and that's what popped out.

Reply:
Just ran `pnpm all` and it cleaned it up... huh..

Reply:
I did run `pnpm install` before this but maybe something borked up along the way.